zk/tests/test_e2e.py

```diff
@@ -0,0 +1,32 @@
+# (C) Datadog, Inc. 2010-2019
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+from __future__ import unicode_literals
+
+import os
+from distutils.version import LooseVersion  # pylint: disable=E0611,E0401
+
+import pytest
+
+from . import common, conftest
```

Comment:
Can you move the metrics in common? It's a more appropriate place than conftest.